tools/base/libboard/board_vc/board_vc.c

```diff
@@ -159,8 +158,7 @@ fpga_result parse_fw_ver(char *buf, char *fw_ver, size_t len)
 	rev = (var >> 24) & 0xff;
 	if ((rev >= 'A') && (rev <= 'Z')) {// range from 'A' to 'Z'
 		snprintf_s_ciii(fw_ver, len, "%c.%u.%u.%u", (char)rev, (var >> 16) & 0xff, (var >> 8) & 0xff, var & 0xff);
```

Comment:
does `snprintf_s_ciii` return an error code? Should we check it?

Reply:
Fixed

retval = snprintf_s_ciii(fw_ver, len, "%c.%u.%u.%u", (char)rev, (var >> 16) & 0xff, (var >> 8) & 0xff, var & 0xff);
		if (retval < 0) {
			FPGA_ERR("error in formatting version" );
			return FPGA_EXCEPTION;
		}